### PR TITLE
[dashing backport] add service clients to ros2node info

### PR DESCRIPTION
**Pull request by dirk-thomas**
Backport of #392.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
> (with green CI)

The referenced PR jobs is already green?

**Comment by ivanpauno** [comment]
> The referenced PR job is already green?

Don't we run again CI against `dashing` branch?
IMO, it's not the same thing.
It's pretty unlikely that this will introduce a failure, as there is not much code in the Dashing branch testing the CLI (except the api, which this is modifying trivially).
So, I don't mind if the re-run is ignored.

**Comment by dirk-thomas** [comment]
> Don't we run again CI against `dashing` branch?

I would say it always depends on the kind of patch. Since this one doesn't have any additional tests and no platform specific logic and passes the PR build I don't see a need to run the CI build on all platforms.

> there is not much code in the Dashing branch testing the CLI (except the api, which this is modifying trivially).

Yeah, the API update is trivial and there is no test covering it on the Dashing branch anyway.

File: ros2node/ros2node/verb/info.py [hunk 1]
```diff
@@ -49,8 +50,13 @@ def main(self, *, args):
                 publishers = get_publisher_info(node=node, remote_node_name=args.node_name)
                 print('  Publishers:')
                 print_names_and_types(publishers)
-                services = get_service_info(node=node, remote_node_name=args.node_name)
-                print('  Services:')
-                print_names_and_types(services)
+                service_servers = get_service_server_info(
+                    node=node, remote_node_name=args.node_name, include_hidden=args.include_hidden)
```
Comment (dirk-thomas) [inline ros2node/ros2node/verb/info.py]: On the `dashing` branch the `info` verb doesn't have the `--include-hidden` option. See #417.

**Pull request merged**